examples/rex_planar_quadrotor_mpc.jl

```diff
@@ -0,0 +1,560 @@
+        )
+        mpc_jDMD = TrackingMPC(model_jDMD_projected, 
+            X_ref, U_ref, Vector(T_sim), Qmpc, Rmpc, Qfmpc; Nt=Nt
+        )
+        X_mpc_nom, U_mpc_nom, T_mpc = simulatewithcontroller(dmodel_real, mpc_nom,  X_ref[1], tf_sim, dt, printrate=false)
+        X_mpc_eDMD,U_mpc_eDMD,T_mpc = simulatewithcontroller(dmodel_real, mpc_eDMD, X_ref[1], tf_sim, dt, printrate=false)
+        X_mpc_jDMD,U_mpc_jDMD,T_mpc = simulatewithcontroller(dmodel_real, mpc_jDMD, X_ref[1], tf_sim, dt, printrate=false)
+
+        err_nom = norm(X_mpc_nom - X_ref_full) / N_sim
+        err_eDMD = norm(X_mpc_eDMD - X_ref_full) / N_sim
+        err_jDMD = norm(X_mpc_jDMD - X_ref_full) / N_sim
+
+        (; err_nom, err_eDMD, err_jDMD) #, t_train_eDMD, t_train_jDMD, num_lqr, num_swingup, nsamples=length(X_train)) end
+    end
+
+    nom_err_avg  = mean(filter(isfinite, map(x->x.err_nom, test_results)))
+    eDMD_err_avg = mean(filter(isfinite, map(x->x.err_eDMD, test_results)))
+    jDMD_err_avg = mean(filter(isfinite, map(x->x.err_jDMD, test_results)))
+    eDMD_success = count(isfinite, map(x->x.err_eDMD, test_results))
+    jDMD_success = count(isfinite, map(x->x.err_jDMD, test_results))
+
+    G = model_jDMD.g
+    kf = model_jDMD.kf
+
+    (;nom_err_avg, eDMD_err_avg, eDMD_success, jDMD_err_avg, jDMD_success, 
+        t_train_eDMD, t_train_jDMD, num_lqr, num_mpc, nsamples=length(X_train), 
+        eDMD_data, jDMD_data, G, kf, dt)
+end
+
+#############################################
+## 
+#############################################
+
+generate_planar_quadrotor_data()
+train_planar_quadrotor_models(10,6)
+num_traj = 2:1:6   # goes singular above 35?
+results = map(num_traj) do N
+    println("\n\nRunning with N = $N")
+    train_planar_quadrotor_models(0,N)
+end
+const PLANAR_QUADROTOR_RESULTS_FILE = joinpath(Problems.DATADIR, "rex_planar_quadrotor_results.jld2")
+jldsave(PLANAR_QUADROTOR_RESULTS_FILE; results)
+
+#############################################
+## Process results
+#############################################
+
+using PGFPlotsX
+results = load(PLANAR_QUADROTOR_RESULTS_FILE)["results"]
+fields = keys(results[1])
+res = Dict(Pair.(fields, map(x->getfield.(results, x), fields)))
+plot(res[:num_mpc], res[:t_train_eDMD])
+plot!(res[:num_mpc], res[:t_train_jDMD])
+
+p = @pgf Axis(
+    {
+        xmajorgrids,
+        ymajorgrids,
+        xlabel = "Number of training samples",
+        ylabel = "Tracking error",
+    },
+    PlotInc({no_marks, "very thick", "black"}, Coordinates(res[:nsamples], res[:nom_err_avg])),
+    PlotInc({no_marks, "very thick", "orange"}, Coordinates(res[:nsamples], res[:eDMD_err_avg])),
+    PlotInc({no_marks, "very thick", "cyan"}, Coordinates(res[:nsamples], res[:jDMD_err_avg])),
+    Legend(["Nominal", "eDMD", "jDMD"])
+)
+pgfsave(joinpath(Problems.FIGDIR, "rex_planar_quadrotor_mpc_test_error.tikz"), p, include_preamble=false)
+
+#############################################
+## Stabilization analysis
```

Comment:
are you actually using the stuff below here? mine was a WIP so if you're not using probably just get rid of it